tools/tensorflow_docs/api_generator/reference_resolver.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+from __future__ import annotations
```

Comment:
This should be the only change you need to make to enable this file to be loaded in Py3.7.